foxy/distribution.yaml

```diff
@@ -1344,11 +1344,15 @@ repositories:
       version: dashing-devel
     status: developed
   mavlink:
+    doc:
+      type: git
+      url: https://github.com/mavlink/mavlink-gbp-release.git
```

Comment:
I guess it makes more sense to point to the upstream repository (https://github.com/mavlink/mavlink), instead of the release repository for documentation. Also, the `source` entry below should probably also be updated to point to https://github.com/mavlink/mavlink.

Reply:
Hello, unfortunately nope, that is special case for 3-rd party library which didn't come with `pakage.xml`. Same done for all other distros.